docs/source/release_notes.rst

```diff
@@ -22,6 +22,7 @@ v0.4.2 Jun 23, 2021
     * Fixes
         * Raise type conversion error in ``Datetime`` logical type (:pr:`1001`)
         * Try collections.abc to avoid deprecation warning (:pr:`1010`)
+        * Add input validation to ww.init_series (:pr: '1017')
```

Comment:
The release notes here are actually from the release that went out yesterday. This line and your name from below should go in the `Future Release` section above (you'll need to remove the two dots `..` in the two places to uncomment it, and it should follow the formatting in this section)

Reply:
I have now put it in the future release!

Reply:
You just need add your own name at the bottom of the  future release section :) everyone else will add their own when they put up PRs

Reply:
Oh okay I added the name now haha